static/js/actions/email.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+export const actionCreatorGenerator = (type: string) => (
+  (args: any) => args === undefined ? { type: type } : { type: type, payload: args }
+);
```

Comment:
seems like this could be useful elsewhere. does it make sense to create `static/js/actions/util.js` (et al) and define this in there?